doc/dune

```
@@ -31,15 +31,12 @@
   (> %{ocaml_version} 4.11))
  (deps
   (package odoc)
+  (universe) ; Benchmark depends on the running programs in the background
```

It's sad to put this on an Mdx rule. The benchmark is only one of the uses of this rule and it doesn't need to be run several time for a given build of Odoc.

Though, it's probably OK for a rule that is run manually and is not part of a more general workflow.

I don't understand the comment, maybe you mean that:
```suggestion
  (universe) ; Benchmark depends on the running time of odoc commands
```

---

I meant that the output of the rule depends on external factors, such as the workload of the computer... which is expressed as a `universe` dependency (the workload being part of the universe!)

---

I'm happy to remove it: Maybe it happens more often that we are just building the doc, than we are doing sequential benchmark runs. One can always do `dune clean` at the expense of recompiling `odoc` itself, which is fine to me.

---

I see. I think it's too specific to be right. For example, it also depends on the temperature and humidity of my room. I prefer the suggested comment.

I'm OK with the `(universe)` deps as I don't see a reason to run this rule twice. The comment will help us remove it if needed in the future.